pkg/broker/catalog.go

```diff
@@ -16,14 +19,19 @@ func (b Broker) Services(ctx context.Context) ([]brokerapi.Service, error) {
 	services := make([]brokerapi.Service, len(clouds))
 
 	for i, cloud := range clouds {
+		// Create a CLI and user-friendly name. Will be displayed in the
```

Comment:
Ambiguous—I read that as create a CLI and also create a user-friendly name. Use a trailing dash to disambiguate or spell out in full:
```suggestion
		// Create a CLI- and user-friendly name. Will be displayed in the
```
```suggestion
		// Create a CLI-friendly and user-friendly name. Will be displayed in the
```

Reply:
You're totally right!